lms/djangoapps/courseware/courses.py

```diff
@@ -552,7 +552,7 @@ def get_course_assignment_date_blocks(course, user, request, num_return=None,
 
 
 @request_cached()
-def get_course_blocks_completion_summary(course_key, user):
+def get_course_blocks_completion_summary(course_key, user, optional=False):
```

Comment:
Please add the explanation of this param to the docstring.

Reply:
done!